openedx/adg/lms/applications/models.py

```diff
@@ -30,25 +35,47 @@ def __str__(self):
         return 'User {user_id}, application status id={id}'.format(user_id=self.user.id, id=self.id)
 
 
+class BusinessLine(TimeStampedModel):
+    """
+    Model to save the business lines
+    """
+    title = models.CharField(verbose_name=_('Title'), max_length=255, unique=True, )
+    logo = models.ImageField(
+        upload_to='business-lines/logos/', null=True, verbose_name=_('Logo'),
+        validators=[FileExtensionValidator(ALLOWED_LOGO_EXTENSIONS), validate_logo_size],
+        help_text=_('Accepted extensions: .png, .jpg, .svg'),
+    )
+    description = models.TextField(verbose_name=_('Description'),)
+
+    class Meta:
+        app_label = 'applications'
+
+    def __str__(self):
+        return '{}'.format(self.title)
+
+
 class UserApplication(TimeStampedModel):
     """
     Model for status of all required parts of user application submission.
     """
-    user = models.OneToOneField(User, related_name='application', on_delete=models.CASCADE, )
+    user = models.OneToOneField(User, related_name='application', on_delete=models.CASCADE, verbose_name=_('User'), )
     city = models.CharField(verbose_name=_('City'), max_length=255, )
+    business_line = models.OneToOneField(BusinessLine, verbose_name=_('Business Line'),
```

Comment:
Why this is **OneToOneField** This must be a **ForiegnKey**, Many applications can have the same business line (Many users can apply for the same business line). @ahmed-faraz46 @zahid-arbisoft @zeshanarifios Right?